scheduler/remote_server_options.py

```diff
@@ -8,39 +8,37 @@
     :copyright: 2019 SuperDARN Canada
 """
 
-from src.utils.general import load_config
-
-
-
-def ascii_encode_dict(data):
-    return dict(map(lambda x: x.encode('ascii'), pair) for pair in data.items())
+import os
+import json
+from dataclasses import dataclass, field
 
 
+@dataclass
 class RemoteServerOptions(object):
     """
     Parses the options from the config file that are relevant to data writing.
 
     """
-    def __init__(self, config_path=None):
-        """
-        Initialize and get configuration options
-
-        Args:
-            config_path (str): path to config file for. Default BOREALISPATH/config/[rad]/[rad]_config.ini
-        """
-        super().__init__()
-
-        # Gather the borealis configuration information
-        raw_config = load_config()
-
-        self._site_id = raw_config["site_id"]
-
-    @property
-    def site_id(self):
-        """
-        Gets the 3 letter radar code of this radar.
-
-        :return:    3 letter radar code
-        :rtype:     str
-        """
-        return self._site_id
+    site_id: str = field(init=False)
```

Comment:
This is a dict, not a string?

Reply:
I think my change will merge conflict with someone else's change to this file.